packages/core/src/services/providerLogs/create.ts

```diff
@@ -51,14 +51,7 @@ export async function createProviderLog(
     const inserts = await trx
       .insert(providerLogs)
       .values({
-        // TODO: Review if wrapping with a `new Date` is necessary.
-        // `generatedAt` is already a `Date` object, so this should work but it doesn't.
-        // I saw this workouround here:
-        // https://github.com/drizzle-team/drizzle-orm/issues/1113#issuecomment-2220076371
-        //
-        // Docs for timestamp with `{ mode: 'date' }`
-        // https://orm.drizzle.team/docs/column-types/pg#timestamp
-        generatedAt: new Date(generatedAt),
+        generatedAt: generatedAt,
```

Comment:
Not necessary?

Reply:
no, my suspicion is that since this happened in a job before, whoever implemented this forgot that properties get serialized into serializable values for the job system to work (so dates get transformed into strings). Since this happens in sync now, no need for this.

Reply:
That was me. Makes sense